docs/how_to/data_visualization/views.md

```diff
@@ -0,0 +1,61 @@
+# How to define views
+
+:::{admonition} What does this guide solve?
+:class: important
+This guide will show you how to define views on your dashboard
+:::
+
+## Overview
+A view is the final output of a dashboard.
+Views are located in the `target` area and can take the form of various visual components. Lumen includes many [view types](../../reference/view/index.md), and is built so that you can easily use components from the [Holoviz ecosystem](https://holoviz.org/), such as a [scatter plot from hvPlot](https://hvplot.holoviz.org/reference/pandas/scatter.html) or an [indicator from Panel](https://panel.holoviz.org/reference/index.html#indicators).
+
+![](../../_static/excalidraw/lumen_dashboard.png)
+
+Below is an example of two views of the same data - a scatter plot and a table.
+
+::::{tab-set}
+:::{tab-item} Specification (YAML)
+:sync: yaml
+``` {code-block} yaml
+:emphasize-lines: 10-14
+sources:
+  penguin_source:
+    type: file
+    tables:
+      penguin_table: https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2020/2020-07-28/penguins.csv
+
+targets:
+  - title: Table
+    source: penguin_source
```

Comment:
I still think we need to explain how a View gets its data and explain about specifying a source/table or pipeline.

Reply:
I'm working on it, which is why I haven't marked this for review yet. :)